src/friendly-iframe-embed.js

```diff
@@ -88,6 +96,25 @@ export function setSrcdocSupportedForTesting(val) {
   srcdocSupported = val;
 }
 
+/**
+ * @param {!Window} win
+ * @return {function(*): !Promise<*>}
+ */
+function getDelayPromise(win) {
+  if (
+    isExperimentOn(win, 'fie-init-chunking') &&
+    getExperimentBranch(win, FIE_INIT_CHUNKING_EXP.id) ===
+      FIE_INIT_CHUNKING_EXP.experiment
+  ) {
+    return (val) =>
```

Comment:
do you think we might use this chunking technique elsewhere with other experiments? If so maybe move it into its own function.

Reply:
Yes, that makes sense. I am in favor of this for now because:
1. it is easier to rollback as it doesn't involve in others;
2. to contain the experiment branching logic for now till we decide which way to go for.

WDYT?

Reply:
SGTM, lets revisit if we do more chunking.